garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,206 @@
+        self._std_output_nonlinearity = std_output_nonlinearity
+        self._std_parameterization = std_parameterization
+        self._hidden_nonlinearity = hidden_nonlinearity
+        self._output_nonlinearity = output_nonlinearity
+
+        # Tranform std arguments to parameterized space
+        self._init_std_param = None
+        self._min_std_param = None
+        self._max_std_param = None
+        if self._std_parameterization == 'exp':
+            self._init_std_param = np.log(init_std)
+            if min_std:
+                self._min_std_param = np.log(min_std)
+            if max_std:
+                self._max_std_param = np.log(max_std)
+        elif self._std_parameterization == 'softplus':
+            self._init_std_param = np.log(np.exp(init_std) - 1)
+            if min_std:
+                self._min_std_param = np.log(np.exp(min_std) - 1)
+            if max_std:
+                self._max_std_param = np.log(np.exp(max_std) - 1)
+        else:
+            raise NotImplementedError
+
+    @property
+    def init_spec(self):
+        """
+        Model specification.
+
+        Positional and named arguments (args and kwargs) which, when passed
+        to `__init__`, will construct this model.
+        """
+        return (), {
+            'name': self._name,
+            'output_dim': self._output_dim,
+            'hidden_sizes': self._hidden_sizes,
+            'learn_std': self._learn_std,
+            'init_std': self._init_std,
+            'adaptive_std': self._adaptive_std,
+            'std_share_network': self._std_share_network,
+            'std_hidden_sizes': self._std_hidden_sizes,
+            'min_std': self._min_std,
+            'max_std': self._max_std,
+            'std_hidden_nonlinearity': self._std_hidden_nonlinearity,
+            'std_output_nonlinearity': self._std_output_nonlinearity,
+            'hidden_nonlinearity': self._hidden_nonlinearity,
+            'output_nonlinearity': self._output_nonlinearity,
+            'std_parameterization': self._std_parameterization,
+        }
+
+    def _build(self, *inputs):
+        action_dim = self._output_dim
+        state_input = inputs[0]
+        with tf.variable_scope("dist_params"):
+            if self._std_share_network:
+                # mean and std networks share an MLP
+                b = np.concatenate([
+                    np.zeros(action_dim),
+                    np.full(action_dim, self._init_std_param)
+                ], axis=0)  # yapf: disable
+                b = tf.constant_initializer(b)
+                mean_std_network = mlp(
+                    state_input,
+                    output_dim=action_dim * 2,
+                    hidden_sizes=self._hidden_sizes,
+                    hidden_nonlinearity=self._hidden_nonlinearity,
+                    output_nonlinearity=self._output_nonlinearity,
+                    output_b_init=b,
+                    name="mean_std_network")
```

Comment:
'mean_std_network'
I don't want to create multiple comments about this, please replace double quotation to single quotation. They are appeared in here and the unittests.

Reply:
please don't resolve comments until the resolved code shows on GitHub